src/index.ts

```diff
@@ -186,12 +187,26 @@ export async function exportActorProfile({
 export async function importActorProfile(
   tarStream: Readable
 ): Promise<Record<string, any>> {
+  console.log('🚀 Starting to process tar stream...')
```

Comment:
Will you make it so `console` can be injected as an argument?

And this line like `console?.log(....)`

that way, by default, this function won't console.log, which is nice outside of the CLI context.

The CLI can inject in `{ console: globalThis.console }` when it calls these functions so that, only in the cli context, this function produces logs. But then there is also a way to opt-out of verbose logging.